ffw/UIRPC.js

```diff
@@ -287,6 +287,33 @@ FFW.UI = FFW.RPCObserver.create(
             }
             break;
           }
+          case 'UI.CancelInteraction':
+          {
+            var targetID = request.params.cancelID;
+            var typeID = request.params.functionID;
+
+            if (typeID === 10 && SDL.InteractionChoicesView.active
+               && (targetID === undefined || targetID === SDL.InteractionChoicesView.cancelID)) {
+              SDL.InteractionChoicesView.deactivate('ABORTED');
+            } else if (typeID === 12 && SDL.AlertPopUp.active
+               && (targetID === undefined || targetID === SDL.AlertPopUp.cancelID)) {
+              SDL.AlertPopUp.deactivate();
```

Comment:
```suggestion
              SDL.AlertPopUp.deactivate("ABORTED");
```
These are terribly inconsistent